src/drt/test/run-ispd.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+# This runs the ISPD 18 & 19 routing benchmarks.  GNU Parallel is used for
+# parallelism over many designs.  This is used by Jenkins for nightly
+# OpenROAD testing.
+
+import argparse
+import fnmatch
+import multiprocessing
+import os
+import shutil
+import stat
+import subprocess
+import sys
+import textwrap
+
+parser = argparse.ArgumentParser(
+    prog="run-ispd",
+    description="Run the ISPD routing benchamrks",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+)
+parser.add_argument(
+    "-d",
+    "--dir",
+    default=os.path.expanduser('~/ispd'),
+    help="Root directory to run under (must have a tests subdir with benchmarks)"
+)
+parser.add_argument(
+    "-t",
+    "--tests",
+    nargs="*",
+    default="*",
+    help="The tests to run. Matched to designs by glob."
+)
+parser.add_argument(
+    "-j",
+    "--jobs",
+    default=4,
+    type=int,
+    help="Number of jobs to run concurrently"
+)
+parser.add_argument(
+    "-p",
+    "--program",
+    default=shutil.which('openroad'),
+    help="Path to openroad to test"
+)
+args = parser.parse_args()
+
```

Comment:
Add a check and error message if the arguments are valid, e.g., does folder exists in `--dir folder` or was the `openroad` bin found.

Reply:
Agreed